themes/default/layouts/partials/registry/package/header.html

```diff
@@ -11,9 +11,8 @@
 {{ end }}
 
 <div class="container">
-    <div class="mb-8 text-blue-600 font-display">
-        <span>&larr;</span>
-        <a href="{{ relref . "/registry" }}" class="hover:underline">All packages</a>
+    <div class="lg:flex mb-6">
+        {{ partial "registry/package/how-to-guides/breadcrumb.html" . }}
```

Comment:
Bit of a nit, but should this partial just be moved to`registry/package/breadcrumb.html`, since it's the same across all pages now (and not specific to `how-to-guides`?

Reply:
It wasn't specific to how to guides previously either (it was used in API docs too), but good call that now's a great time to move it, will do!